docs/integrations/sigrid-api-documentation.md

```diff
@@ -541,6 +505,38 @@ com/rest/analysis-results/api/v1/objectives/{customer}`) is as follows:
 
 </details>
 
+#### Retrieving objectives for a system
+
+    GET https://sigrid-says.com/rest/analysis-results/api/v1/objectives/{customer}/{system}/config
+    
+This end point returns the following response structure:
+
+    {
+      "MAINTAINABILITY": 4.0,
+      "NEW_CODE_QUALITY": 3.5,
+      "OSH_MAX_SEVERITY": "LOW",
+      "TEST_CODE_RATIO": 0.8
+    }
+
+In Sigrid, a system always has at most one objective per type. So in the above example, the 
```

Comment:
I probably suggested this text myself, but I'm now wondering whether it should say: "In Sigrid, a system always has at most one _target_ per type".

Reply:
I rewrote the entire section a bit. I'm now separating the concept of precedence rules existing from the concrete examples. And yes, I'm intentionally not explaining the situation where you have multiple conditional objectives, I find that makes it a bit too complicated.